arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -1151,1137 +1151,4913 @@
+          "description": "The correlation properties."
+        },
+        "inputsLink": {
+          "$ref": "#/definitions/ContentLink",
+          "readOnly": true,
+          "description": "Gets the link to inputs."
+        },
+        "outputsLink": {
+          "$ref": "#/definitions/ContentLink",
+          "readOnly": true,
+          "description": "Gets the link to outputs."
+        },
+        "trackedProperties": {
+          "$ref": "#/definitions/Object",
+          "readOnly": true,
+          "description": "Gets the tracked properties."
+        }
+      },
+      "description": "The workflow run action properties."
+    },
+    "WorkflowRunActionFilter": {
+      "type": "object",
+      "properties": {
+        "status": {
+          "$ref": "#/definitions/WorkflowStatus",
+          "description": "The status of workflow run action."
+        }
+      },
+      "description": "The workflow run action filter."
+    },
+    "WorkflowRunActionListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WorkflowRunAction"
+          },
+          "description": "A list of workflow run actions."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "The list of workflow run actions."
+    },
+    "SkuName": {
+      "type": "string",
+      "enum": [
+        "NotSpecified",
+        "Free",
+        "Shared",
+        "Basic",
+        "Standard",
+        "Premium"
+      ],
+      "x-ms-enum": {
+        "name": "SkuName",
+        "modelAsString": false
+      },
+      "description": "The sku name."
+    },
+    "WorkflowState": {
+      "type": "string",
+      "enum": [
+        "NotSpecified",
+        "Completed",
+        "Enabled",
+        "Disabled",
+        "Deleted",
+        "Suspended"
+      ],
+      "x-ms-enum": {
+        "name": "WorkflowState",
+        "modelAsString": false
+      }
+    },
+    "WorkflowStatus": {
+      "type": "string",
+      "enum": [
+        "NotSpecified",
+        "Paused",
+        "Running",
+        "Waiting",
+        "Succeeded",
+        "Skipped",
+        "Suspended",
+        "Cancelled",
+        "Failed",
+        "Faulted",
+        "TimedOut",
+        "Aborted",
+        "Ignored"
+      ],
+      "x-ms-enum": {
+        "name": "WorkflowStatus",
+        "modelAsString": false
+      }
+    },
+    "ParameterType": {
+      "type": "string",
+      "enum": [
+        "NotSpecified",
+        "String",
+        "SecureString",
+        "Int",
+        "Float",
+        "Bool",
+        "Array",
+        "Object",
+        "SecureObject"
+      ],
+      "x-ms-enum": {
+        "name": "ParameterType",
+        "modelAsString": false
+      }
+    },
+    "KeyType": {
+      "type": "string",
+      "enum": [
+        "NotSpecified",
```

Comment:
Looks like this value was already there.  @amarzavery did you mean a different enum?